src/devices/PiJuice/PiJuice.cs

```diff
@@ -63,57 +63,53 @@ public Version GetFirmwareVerion()
             var response = ReadCommand(PiJuiceCommand.FirmwareVersion, 2);
 
             var major_version = response[0] >> 4;
-            var minor_version = (response[0] << 4 & 0xf0) >> 4;
+            var minor_version = response[0] & 0x0F;
 
             return new Version(major_version, minor_version);
         }
 
         /// <summary>
-        /// Restore PiJuice to its default settings
+        /// Restore PiJuice to its default settings (https://github.com/PiSupply/PiJuice/blob/02c129c3aa884934c368936c9363df90690148a8/Software/Source/pijuice.py#L1531)
```

Comment:
just the url as a comment before the WriteCommand, it doesn't need to be in the description of what the function is doing